### DISCUSSION TITLE
Phishing warning - option to disable "[Possible phishing attempt]" in subject line

### DISCUSSION THREAD

**Discussion by alexhk90** [simple-login/app#2182] [Feature Requests]
There is already a warning in the email itself "This email failed anti-phishing checks when it was received by SimpleLogin, be careful with its content. More info on anti-phishing measure" and in ProtonMail "This email has failed its domain's authentication requirements. It may be spoofed or improperly forwarded. Learn more" - I consider this to be more than sufficient, so would like the option to disable the recently added "[Possible phishing attempt]" in subject line as it is superfluous and there are too many false positives for it to be useful. This could be opt-out with a large warning so users know they are opting to take on some additional risk by doing this.

**Comment by soulstenance** [comment]
I agree. Also solving the false positives problem would be really nice and make this less of an issue. Like, I send a support email using my reverse alias and the get a response back to that alias - in what universe could that possibly be phishing? It's really annoying and makes me worried that the support I'm legitimately trying to reach may not even get a hold of me if Simple Login starts blocking the emails altogether.

**Comment by benjuade** [comment]
We need this.

**Comment by Paleogirl2003** [comment]
Please make the subject line part optional in settings. It's very annoying, the red banner in the email itself is sufficient. There are several companies I've contacted to fix their dmarc and they won't so it is distracting when I know they are legit. I may try to see if I can make a sieve in proton to strip it but I shouldn't have to.

**Comment by huntfx** [comment]
The feature itself worked fine for the most part - until I set up an auto forward from my work email as a backup.

Because anything coming out of work gets a footer attached, literally every single email not from @work.com gets flagged for phishing, which makes it a bit useless as a backup.

Doesn't look like sieve filters allow editing subject lines, so I'll have to make a new proton address instead and totally bypass SL. It's just a bit annoying to be so powerless about one specific feature when SL is otherwise great.
